pkg/apis/pingcap/v1alpha1/validation/validation.go

```diff
@@ -525,6 +527,19 @@ func validatePathNoBacksteps(targetPath string, fldPath *field.Path) field.Error
 	return allErrs
 }
 
+func validateTimeDurationStr(timeStr *string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if timeStr != nil {
+		d, err := time.ParseDuration(*timeStr)
+		if err != nil {
+			allErrs = append(allErrs, field.Invalid(fldPath, timeStr, "mush be a valid Go time duration string"))
+		} else if d <= 0 {
```

Comment:
Is it better to mark the value invalid if it is less than `3m`? In case that users set `3` or something similar.

Reply:
It will return error `time: missing unit in duration 3` for `time.ParseDuration`. I think we don't need that